po/ru.po

```diff
@@ -10,6 +15,70 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+
+#: src/Utils.vala:11 src/Utils.vala:31
+#, c-format
+msgid "Gif is Up at %s!"
+msgstr ""
+
+#: src/Utils.vala:16 src/Utils.vala:36
+msgid "Check if all fields have sane values and try again."
+msgstr ""
+
+#: src/Utils.vala:44
+msgid "Status"
+msgstr ""
+
+#: src/Utils.vala:121
+msgid "Is a file selected?"
+msgstr ""
+
+#: src/Window.vala:23
+msgid "Gifup"
+msgstr ""
+
+#: src/Window.vala:44
+msgid "Open your favourite file"
+msgstr ""
+
+#: src/Window.vala:62
+msgid "Basic Options"
+msgstr ""
+
+#: src/Window.vala:64
+msgid "Advanced Options"
+msgstr ""
+
+#: src/Window.vala:79
+msgid "Make GIF!"
+msgstr ""
+
+#: src/Window.vala:91
+msgid "Convert Complete Video"
+msgstr ""
+
+#: src/Widgets/AdvanceOptions.vala:22
+msgid "Gif height:"
+msgstr ""
+
+#: src/Widgets/AdvanceOptions.vala:29
+msgid "Gif width:"
+msgstr ""
+
+#: src/Widgets/AdvanceOptions.vala:35
+msgid "Frames Per Seconds:"
+msgstr ""
+
+#: src/Widgets/BasicOptions.vala:21
+msgid "Start Time :"
+msgstr ""
+
+#: src/Widgets/BasicOptions.vala:25
+msgid "End Time :"
+msgstr ""
+=======
```

Comment:
```suggestion
```

I guess this is the biggest reason the CI still fails, maybe it appeared while resolving conflicts.

Reply:
Whoops! You're right I didn't see that.